cms/envs/common.py

```diff
@@ -2287,6 +2287,9 @@
 ##### LOGISTRATION RATE LIMIT SETTINGS #####
 LOGISTRATION_RATELIMIT_RATE = '100/5m'
 
+##### REGISTRATION RATE LIMIT SETTINGS #####
+REGISTRATION_RATELIMIT = '30/7d'
```

Comment:
You might need to add this into `production.py` for both LMS and CMS to control this value through remote config in future if needed. In this we don't need to change anything in platform and just assign the new value in remote config repo. For example https://github.com/edx/edx-platform/blob/883d42ff44e03b0bfba63764d38bfeae13717b7e/lms/envs/production.py#L604